packages/headless/src/auth.ts

```diff
@@ -56,3 +56,26 @@ export function useResetPassword({
     mutationKey: [MutationKey.ResetPassword],
   });
 }
+
+export function useVerifyEmail({
+  onSuccess,
+  onError,
+}: {
+  onSuccess: OnSuccess;
+  onError: OnError;
+}) {
+  const atlas = useAtlas();
+  const verifyEmail = useCallback(
+    (token: string) => {
+      return atlas.auth.verifyEmail(token);
+    },
+    [atlas]
```

Comment:
This is not the right deps. 

```suggestion
    [atlas.auth]
```